issue 1905/support for multiple images to kind load (#1920)
kubernetes-sigs/kind: site/content/docs/user/quick-start.md

If the flag `--name` is not specified, kind will use the default cluster
context name `kind` and delete that cluster.

## Loading Images Into Your Cluster

> Docker images can be loaded into your cluster nodes by specifying their names as comma-separated:

Reviewer: ```suggestion
Docker images can be loaded into your cluster nodes by specifying their names as a comma-separated list:
```

If the flag `--name` is not specified, kind will use the default cluster
context name `kind` and delete that cluster.

> ## Loading Images Into Your Cluster

Reviewer 2: This will break inbound heading links. The title change doesn't seem worth it in that light.

Author: Ahh, good point. Although I just found that there is only one inbound heading link which I guess we can simply change to point to `Loading images into your cluster` instead.
In my opinion, I guess we can go forwards with changing that inbound link as its a tiny change anyways but let me know what do you think :)

PS: [under this heading only is that one inbound heading link](https://github.com/kubernetes-sigs/kind/blob/master/site/content/docs/user/private-registries.md#pull-to-the-host-and-side-load)

Reviewer 2: > Ahh, good point. Although I just found that there is only one inbound heading link which I guess we can simply change to point to Loading images into your cluster instead.

Er my concern is not the links within the site, but all the places we've handed out this link to users.

On most pages I'd agree, but this page has too much content (before we started making individual guides) and has been around a while, I'm certain there are many more links to this floating around out there 😅

Author: Coolz, sounds legit. I ll revert this one.

## Loading an Image Into Your Cluster

> Docker images can be loaded into your cluster nodes by specifying their names as a space-separated list:

Reviewer 2: the "space-seperate list" bit should also be dropped here
```suggestion
Docker images can be loaded into your cluster nodes with:
```
the example and `--help` text should be sufficient to define the format (and again, it may not be spaces, it's actually controlled by `$IFS` in most shells, and in a calling program it will be an argument array)

Author: Aah, forgot to update this one :see_no_evil
